internal/cli/atlas/backup/compliancepolicy/setup.go

```diff
@@ -0,0 +1,127 @@
+)
+
+type SetupOpts struct {
+	cli.GlobalOpts
+	cli.WatchOpts
+	policy  *atlasv2.DataProtectionSettings
+	store   store.CompliancePolicy
+	fs      afero.Fs
+	path    string
+	confirm bool
+}
+
+var setupTemplate = `
+Your backup compliance policy has been set up with the following configuration:
+
+Project:	{{.ProjectId}}
+Authorized e-mail:	{{.AuthorizedEmail}}
+Copy protection enabled:	{{.CopyProtectionEnabled}}
+Encryption at rest enabled:	{{.EncryptionAtRestEnabled}}
+Point-in-Time restores enabled:	{{.PitEnabled}}
+Restore window days:	{{.RestoreWindowDays}}
+
+POLICIES
+ID	FREQUENCY INTERVAL	FREQUENCY TYPE	RETENTION
+{{- range .ScheduledPolicyItems}}
+{{.Id}}	{{if eq .FrequencyType "hourly"}}{{.FrequencyInterval}}{{else}}-{{end}}	{{.FrequencyType}}	{{.RetentionValue}} {{.RetentionUnit}}
+{{- end}}
+{{if .OnDemandPolicyItem}}{{.OnDemandPolicyItem.Id}}	-	{{.OnDemandPolicyItem.FrequencyType}}	{{.OnDemandPolicyItem.RetentionValue}} {{.OnDemandPolicyItem.RetentionUnit}}{{end}}
+`
+
+func (opts *SetupOpts) initStore(ctx context.Context) func() error {
+	return func() error {
+		var err error
+		opts.store, err = store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx))
+		return err
+	}
+}
+
+func (opts *SetupOpts) setupWatcher() (bool, error) {
+	res, err := opts.store.DescribeCompliancePolicy(opts.ConfigProjectID())
+	opts.policy = res
+	if err != nil {
+		return false, err
+	}
+	if res.GetState() == "" {
+		return false, errors.New("could not access State field")
+	}
+	return (res.GetState() == active), nil
+}
+
+func (opts *SetupOpts) Run() error {
+	opts.policy.SetProjectId(opts.ConfigProjectID())
+
+	_, err := opts.store.UpdateCompliancePolicy(opts.ConfigProjectID(), opts.policy)
+	if err != nil {
+		return err
+	}
+	if err := opts.Watch(opts.setupWatcher); err != nil {
+		return err
+	}
+
+	return opts.Print(opts.policy)
+}
+
+func SetupBuilder() *cobra.Command {
+	opts := &SetupOpts{
+		policy: new(atlasv2.DataProtectionSettings),
+		fs:     afero.NewOsFs(),
+	}
+	use := "setup"
+	cmd := &cobra.Command{
+		Use:     use,
+		Aliases: cli.GenerateAliases(use),
+		Short:   "Setup the backup compliance policy for your project with a configuration file.",
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return opts.PreRunE(
+				opts.ValidateProjectID,
+				opts.initStore(cmd.Context()),
+				opts.InitOutput(cmd.OutOrStdout(), setupTemplate),
+			)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := file.Load(opts.fs, opts.path, opts.policy); err != nil {
+				return err
+			}
+			return opts.Run()
+		},
+	}
+
+	cmd.Flags().StringVar(&opts.ProjectID, flag.ProjectID, "", usage.ProjectID)
+	cmd.Flags().StringVarP(&opts.Output, flag.Output, flag.OutputShort, "", usage.FormatOut)
+	_ = cmd.RegisterFlagCompletionFunc(flag.Output, opts.AutoCompleteOutputFlag())
+	cmd.Flags().StringVarP(&opts.path, flag.File, flag.FileShort, "", usage.BackupCompliancePolicyFile)
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&opts.path, flag.File, flag.FileShort, "", usage.BackupCompliancePolicyFile)
	cmd.Flags().BoolVarP(&opts.EnableWatch, flag.EnableWatch, flag.EnableWatchShort, false, usage.EnableWatch)
```

Reply:
What should we print if the user does not use `--watch`? I think the current output message would be misleading since we can't say for sure that it's been set up, right?

Reply:
locking the terminal should always be opt-in, the question then becomes, what's the default state, `ENABLING`?
when watch is true and it finishes (what happens if I ctrl+c in the middle?) 
then you have
>Your backup compliance policy has been set up with the following configuration

Default should be 
>Enabling your backup compliance policy with the following configuration

Reply:
>what's the default state, `ENABLING`?

Yes.

I'll change it to what you proposed.